src/main/java/org/isf/sms/service/SmsSenderOperations.java

```diff
@@ -21,34 +21,39 @@
  */
 package org.isf.sms.service;
 
-import java.util.List;
```

Comment:
Please honor the projects ordering of imports; see https://github.com/flowable/flowable-engine/blob/main/ide-settings/eclipse/Preferences-Java-CodeStyle-OrganizeImports.importorder

Reply:
Sorry forgot to add the [OpenHospital-code-style-configuration.xml](https://github.com/informatici/openhospital-core/blob/develop/.ide-settings/idea/OpenHospital-code-style-configuration.xml) , should be fixed